kommunicate/src/main/java/io/kommunicate/models/KmAppSettingModel.java

```diff
@@ -116,5 +117,9 @@ public int getBotMessageDelayInterval() {
         public void setBotMessageDelayInterval(int botMessageDelayInterval) {
             this.botMessageDelayInterval = botMessageDelayInterval;
         }
+
+        public boolean isSingleThreaded() {
```

Comment:
wrong method. if value is x, then method should isX
in this case, value is isX and method is also isX

Reply:
fixed